owl/SOMA-IO.owl

```diff
@@ -83,38 +554,454 @@
+                    </owl:Restriction>
+                </owl:intersectionOf>
+            </owl:Class>
+        </owl:equivalentClass>
+        <rdfs:comment xml:lang="en">Any Text that adheres to some rules that are in any way more specific than natural language and that cannot be made sense of without knowing said rules.</rdfs:comment>
+        <rdfs:label xml:lang="en">Structured Text</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#SymbolicReasoner -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#SymbolicReasoner">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Reasoner"/>
+        <rdfs:comment xml:lang="en">A Symbolic Reasoner, is a piece of software able to infer logical consequences from a set of asserted facts or axioms.
+
+Source: https://en.wikipedia.org/wiki/Semantic_reasoner</rdfs:comment>
+        <rdfs:label xml:lang="en">Symbolic reasoner</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#System -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#System">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#SocialObject"/>
+        <rdfs:comment xml:lang="en">A system is a group of interacting or interrelated elements that act according to a set of rules to form a unified whole.
+
+From Wikipedia: https://en.wikipedia.org/wiki/System</rdfs:comment>
+        <rdfs:label xml:lang="en">System</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#Text -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#Text">
+        <owl:equivalentClass>
+            <owl:Class>
+                <owl:intersectionOf rdf:parseType="Collection">
+                    <rdf:Description rdf:about="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#InformationObject"/>
+                    <owl:Restriction>
+                        <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#isGivenMeaningBy"/>
+                        <owl:someValuesFrom rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Language"/>
+                    </owl:Restriction>
+                </owl:intersectionOf>
+            </owl:Class>
+        </owl:equivalentClass>
+        <rdfs:comment xml:lang="en">Any Information Object defined using a Language that, when realized through a symbolic writing system, can be read and made sense of.
+
+One may define Text as anything that can be made sense of, including e.g., Speech or Paintings.
+
+However, the argument can be made that Speech or Paintings are not considered Text as they cannot be EXACTLY realized by a symbolic writing system: Speech may loose punctuation, Paintings their original appearance.
+On the other hand, this might not be true as both could be encoded in a binary format that can be interpreted using a language (eg., mp3, png).</rdfs:comment>
```

Comment:
"Text" in semiotics(?) has a very broad definition, which would include speech, film, and pretty much anything as long as you can claim there exists some system by which to interpret it.